certbot-apache/certbot_apache/parser.py

```diff
@@ -28,7 +28,7 @@ class ApacheParser(object):
     arg_var_interpreter = re.compile(r"\$\{[^ \}]*}")
     fnmatch_chars = set(["*", "?", "\\", "[", "]"])
 
-    def __init__(self, aug, root, vhostroot, version=(2, 4)):
+    def __init__(self, aug, root, vhostroot, version=(2, 4), extra_path=False):
```

Comment:
Both of these are now fixed, I also added test coverage for the non-symlinked files in sites-enabled.